tests/e2e/standard/test_base.py

```diff
@@ -171,3 +172,25 @@ def test_add_and_remove_tags(self, container: MetadataContainer):
             remaining_tag1,
             remaining_tag2,
         }
+
+    @pytest.mark.parametrize("container", AVAILABLE_CONTAINERS, indirect=True)
```

Comment:
We should also test something like filtering by group tags with `fetch_runs_table` like we do for `sys/tags`.